src/controllers/user.js

```diff
@@ -138,6 +128,45 @@ const updatePassword = async (req, res) => {
   }
 };
 
+const updateAddress = async (req, res) => {
+  try {
+    const { addressID, operation } = req.params;
+    const User = await UserModel.findById(req.user.userId);
+    let message;
+    if (operation === 'add') {
+      if (req.body.address !== null && req.body.address !== undefined) {
```

Comment:
We can improve this later to extract the address directly from the `req.body`, and not from the `req.body.address`.